util/exec_test.go

```diff
@@ -168,3 +168,50 @@ func TestParallelExecute(t *testing.T) {
 	})
 
 }
+
+func TestFlags(t *testing.T) {
+	type testCase struct {
+		Name string
+
+		Cmd any
+
+		ExpectedArgs []string
+	}
+
+	validate := func(t *testing.T, tc *testCase) {
+		t.Run(tc.Name, func(t *testing.T) {
+			actualArgs := Flags(tc.Cmd)
+
+			assert.Equal(t, tc.ExpectedArgs, actualArgs)
+		})
+	}
+
+	validate(t, &testCase{
+		Name: "It returns arguments defined with tag `long`",
```

Comment:
nit: would call just `Struct Command` and then the other example `Pointer Struct Command`